src/components/InstancesList/InstancesList.js

```diff
@@ -225,6 +232,21 @@ class InstancesView extends React.Component {
 
     return (
       <Fragment>
+        <IfPermission perm="ui-inventory.instance.create">
+          <Button
+            buttonStyle="dropdownItem"
+            id="clickable-newinventory"
+            onClick={buildOnClickHandler(this.openCreateInstance)}
+          >
+            <Icon
+              icon="plus-sign"
```

Comment:
IMHO, icon could be removed, it contradicts folio style, check bad example https://ux.folio.org/docs/guidelines/components/button/

Reply:
it's requirement

Reply:
and it's dropdown item

Reply:
so it's weird that requirement doesn't follow style guidelines or vice versa

Reply:
style guide is about single button and not about dropdown menu